tools/tests/bundle-ignore-files.js

```diff
@@ -0,0 +1,22 @@
+var _ = require('underscore');
+var selftest = require('../tool-testing/selftest.js');
+var Sandbox = selftest.Sandbox;
+var files = require('../fs/files.js');
+var bundler = require('../isobuild/bundle.js');
```

Comment:
Should this be `../isobuild/bundle.js`?

You can run this test locally with `./meteor self-test "bundle-ignore-files"`